test/integration/utils/test-utils.ts

```diff
@@ -0,0 +1,11 @@
+import { HardhatEthersSigner } from "@nomicfoundation/hardhat-ethers/signers";
+import { impersonate, randomAddress } from "lib";
+import { ether } from "../../../lib/units";
+
+export async function getRandomSigners(amount: number): Promise<HardhatEthersSigner[]> {
+  const signers = [];
+  for (let i = 0; i < amount; i++) {
+    signers.push(await impersonate(randomAddress(), ether("1")));
```

Comment:
```suggestion
    signers.push(await impersonate(randomAddress(), ether("10000")));
```
It's ok to have some big amount, may be used for deposits and some funding

Reply:
fixed